Makefile

```diff
@@ -2,7 +2,8 @@
 PREFIX?=$(shell pwd)
 
 # Used to populate version variable in main package.
-GO_LDFLAGS=-ldflags "-X `go list ./version`.Version `git describe --match 'v[0-9]*' --dirty='.m' --always`"
+GIT_BRANCH=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
```

Comment:
Does this conflict with the `GIT_BRANCH` declared below? Should we call this `VERSION`?